### PR TITLE
fix: use glob to select files to move

### PR DESCRIPTION
**Pull request by ascorbic**
Previously we used the routes manifest to choose files to move, but this didn't include all files. Switched to glob instead, and add check for whether default locale exists before trying to move it.

Test plan:
Run a build, and check that all html and json files in `.next/server/pages` are moved up into `.next`, and are listed in the `static-manifest.json`. This excludes `500.html`, `404.html` and all `*.nft.json` files

### CODE REVIEW

**Comment by tiffanosaurus** [comment]
I ran `npm run build:demo` but not seeing a `static-manifest.json`. Is there a step I'm missing?

**Comment by ascorbic** [comment]
Oh, yes! You need to set the env var, because I stopped it being enabled by default. So run `EXPERIMENTAL_MOVE_STATIC_PAGES=1 npm run build:demo`

**Comment by tiffanosaurus** [comment]
> Oh, yes! You need to set the env var, because I stopped it being enabled by default. So run `EXPERIMENTAL_MOVE_STATIC_PAGES=1 npm run build:demo`

hmm still not seeing it:

CleanShot 2021-11-05 at 11 55 23@2x

**Comment by ascorbic** [comment]
Oh wait! It needs the plugin to run! So run `EXPERIMENTAL_MOVE_STATIC_PAGES=1 ntl build`!

**Comment by tiffanosaurus** [comment]
ohhhhhhhhh of course that makes sense lol.

everything looks good!

**Review by tiffanosaurus — APPROVED** [review]

File: src/helpers/config.js [hunk 1]
```diff
@@ -150,6 +150,7 @@ exports.configureHandlerFunctions = ({ netlifyConfig, publish, ignore = [] }) =>
       `${publish}/*.json`,
       `${publish}/BUILD_ID`,
       `${publish}/static/chunks/webpack-middleware*.js`,
+      `!${publish}/server/**/*.js.nft.json`,
```
[the remark this answers is not in the document]
Comment (ascorbic) [reply src/helpers/config.js:153]: We're missing a trick there

**Review by mraerino — APPROVED** [review]
lgtm

**Comment by ascorbic** [comment]
I'm going to merge this and fix the WIndows path issue in a followup up PR

File: src/helpers/config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (getTobiasNielsen) [reply src/helpers/config.js:153]: @ascorbic Do you support *.nft.json files currently? I'm wondering what trick you are referencing to.

[the remark this answers is not in the document]
Comment (ascorbic) [reply src/helpers/config.js:153]: @getTobiasNielsen Oh, it's just a joke about nfts! We don't currently use the json files directly, though we do use nft during the bundle stage.

[the remark this answers is not in the document]
Comment (getTobiasNielsen) [reply src/helpers/config.js:153]: Does this mean that you handle them like vercel does or are there a difference atm?

[the remark this answers is not in the document]
Comment (ascorbic) [reply src/helpers/config.js:153]: Mostly the same. Vercel does use the JSON files if they exist, but we don't yet and instead do the tracing when bundling, like Vercel does when the JSON files aren't available.

**Pull request merged**